CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ project(ledger-core)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 option(TARGET_JNI "Indicates wheter or not the toolchain must build for JNI or not" OFF)
 option(BUILD_TESTS "Indicates wheter or not the toolchain must build the test or not" ON)
+option(TESTS_UPDATE_HTTP_CACHE "Update HTTP cache for tests" OFF)
+option(TESTS_RUN_WITHOUT_HTTP_CACHE "Don't use cached HTTP responses" OFF)
```

Comment:
(optional) It looks strange to me that each option is `ON` for adding a behavior and this one is for `ON` removing a behavior. I would propose to rename the cmake variable to  `TESTS_USE_HTTP_CACHE`.

Reply:
Done

Reply:
ok !